README.md

```diff
@@ -5,7 +5,7 @@ This repo provides a space for Flutter's UXR team to collaborate with Flutter co
 How to use this repo:
 * [Issues](https://github.com/flutter/uxr/issues) are for tracking tasks in an active research study we conduct in public. Resolutions of issues usually are achieved by changes made to files in the repo (including the wiki). Anyone can post in this section.
 * [Discussions](https://github.com/flutter/uxr/discussions) are for having open-ended conversations about a research topic. There is no expectation that those conversations must reach a resolution nor closure. Anyone can post in this section.
-* [Wiki](https://github.com/flutter/uxr/wiki) is used to host public research documents that we do not expect frequent changes or do not expect community contributons. Only maintainers of this repo can edit the wiki section.
+* [Wiki](./docs/README.md) is used to host public research documents that we do not expect frequent changes or do not expect community contributons. Only maintainers of this repo can edit the wiki section.
```

Comment:
Not a Wiki :)

Reply:
I have a follow up PR, I'll fix these in that. :)